docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -69,7 +69,7 @@ security setup etc.). As the information contained in the DTR may be sensitive a
 every data provider must offer his own DTR as an EDC Data Asset. While it is only mandatory to implement the GET
 endpoints
 as specified in
-the [Development View](API%20AAS%20Registry/dotaas-part-2-http-rest-registry-service-specification.info.mdx),
+the [Development View](../../../../openApi/dt/registry-service-ssp002-openapi.yaml),
```

Comment:
Hi @arnoweiss, I just saw, that you updated the relative path to an absolute path for the Submodel API. Maybe that one should be changes to an absolute path as well?